bridge/engine/EngineMixer.cpp

```diff
@@ -452,6 +491,17 @@ void EngineMixer::removeStream(EngineVideoStream* engineVideoStream)
         }
     }
 
+    for (auto& ssrcOutboundContextPair : engineVideoStream->ssrcOutboundContexts)
+    {
+        if (!ssrcOutboundContextPair.second.markedForDeletion)
+        {
+            engineVideoStream->transport.getJobQueue().addJob<RemovePacketCacheJob>(*this,
```

Comment:
When we remove video stream we have to remove packet cache as well, otherwise it will be removed only when the Mixer is deleted. This because `_videoPacketCaches` in `Mixer`  is deleted when message `FreeVideoPacketCache` is received only